### ISSUE TITLE
Sugar Babies App and Webview don't match up when adressing cloudfare.

### ISSUE THREAD

**Issue by HaarisAhmed2001** [tachiyomiorg/tachiyomi-extensions#15411]
### Source information and language

Sugar Babies English 1.4.30

### Steps to reproduce

1. Press either the latest updates page or just the extension itself to access the searching.
2. wait to loud
3. blocked by cloudfare
4. try webview option
5. wait to pass cloudfare
6. once it passes close webview
7. reload latest page or popular page

### Expected behavior

The source should've shown the results that were equivalent to what was shown in the WebView, whatever was the latest or the most popular that was displayed within the WebView should appear in the source's pages.

### Actual behavior

The WebView showed the expected results but the source page on tachiyomi kept showing the cloudfare error, I attempted all the other methods to get around the cloudfare but it didn't budge, this didn't happen pre extension update. 

### Tachiyomi version

0.14.5

### Android version

Android 11

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.14.5](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.
